### PR TITLE
Create the option to not have a top level object

### PR DESCRIPTION
**Pull request by DanielHabib**
Create the option to not have a top level object.

We will have a `ToplessResponse` that calls this ToplessOutput

### CODE REVIEW

File: src/ResponseBody.php [hunk 1]
```diff
@@ -38,6 +38,15 @@ public function getOutput()
     {
         $response = [];
 
+        if (count($this->members) == 1) {
+            $member = $this->members[0];
```
Comment (kayladnls) [inline src/ResponseBody.php]: this logic feels really dirty and a little clunky.

Comment (kayladnls) [inline src/ResponseBody.php]: What happens if you have 2 members that are both CustomResult objects?

File: src/ResponseBody.php [hunk 2]
```diff
@@ -37,6 +37,12 @@ public function getMembers()
     public function getOutput()
     {
         $response = [];
+        reset($this->members);
+        $member = current($this->getMembers());
+
+        if ($member->getTopLevelName() === '') {
```
Comment (kayladnls) [inline src/ResponseBody.php]: would rather check ` if ($member instanceof CustomResult){` Seems a lot stronger than checking for empty string.

**Comment by DanielHabib** [comment]
@kayladnls Adjusted the check!

**Comment by kayladnls** [comment]
:+1:

**Pull request merged**